rebasehelper/specfile.py

```diff
@@ -1528,10 +1532,11 @@ def run_spec_hooks(self, spec_file, rebase_spec_file, **kwargs):
         :param kwargs: Keyword arguments from Application instance
         """
         for name, spec_hook in six.iteritems(self.spec_hooks):
-            categories = spec_hook.get_categories()
-            if not categories or spec_file.category in categories:
-                logger.info("Running '%s' spec hook", name)
-                spec_hook.run(spec_file, rebase_spec_file, **kwargs)
+            if spec_hook.__name__ not in kwargs["disable_spec_hook"]:
```

Comment:
You could reverse the condition and just `continue` if it passes. It would prevent unnecessary indentation.